packages/babel-types/src/definitions/jsx.ts

```diff
@@ -42,7 +42,7 @@ defineType("JSXClosingElement", {
 });
 
 defineType("JSXElement", {
-  builder: ["openingElement", "closingElement", "children", "selfClosing"],
```

Comment:
I think this change should be behind the `BABEL_8_BREAKING` flag, because a different shape is returned from `t.JSXElement`.

For example, see
https://github.com/babel/babel/blob/2c7e9f79bb8c28c73ecc52c53fa1bc3a25c21b63/packages/babel-types/src/definitions/core.ts#L457-L462

Reply:
That's right. I'll fix it later.